src/models/subject.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Subject extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    // eslint-disable-next-line no-unused-vars
+    static associate(models) {
+      // define association here
+    }
+  }
+  Subject.init(
+    {
+      subjectName: DataTypes.STRING,
```

Comment:
Нужно добавить allowNull:false и unique:true в модели, так как поле обязательное и название предмета уникальное
```suggestion
      subjectName: {
        type: DataTypes.STRING,
        allowNull: false,
        unique: true,
      },
```

Reply:
@BauyrjanSeidahmet Переправил